### ISSUE TITLE
Static files are missing from deployed pyk API docs

### ISSUE THREAD

**Issue by tothtamas28** [runtimeverification/k#4393]
Related:
* #4379

In particular some CSS and JS files, and as a consequence the theme is missing from https://kframework.org/pyk/.

What should be deployed on GH Pages:

```
pyk
├── _modules
├── _sources
├── _static
└── api
```

What's actually deployed:

```
pyk
└── api
```

The content is lost during the Jekyll build phase. A failed attempt to fix this was:
* #4392

**Comment by tothtamas28** [comment]
`_config.yml` does not seem to be read during build: https://github.com/runtimeverification/k/actions/runs/9254162277/job/25455340538#step:4:11

As the file is not pushed to the `gh-pages` branch: https://github.com/runtimeverification/k/tree/gh-pages

@yiyi-wang-rv, do we even use Jekyll for https://kframework.org/? If not, I'll just delete `_config.yml` and disable the build.

**Comment by yiyi-wang-rv** [comment]
@tothtamas28 you are right. We are not using jekyll.

**Comment by yiyi-wang-rv** [comment]
To add "pyk" directory to our website, we need to modify this script 

https://github.com/runtimeverification/k/blob/master/web/build-html.js

I am not available now. @asavienko could you please help take a look at this? 

Thank you

**Comment by tothtamas28** [comment]
> we need to modify this script

It is probably not necessary at this point, the pyk API documentation uses Sphinx and has its own build steps: https://github.com/runtimeverification/k/blob/master/.github/workflows/release.yml#L466-L470

**Comment by tothtamas28** [comment]
Fixed in #4395
